components/cdc/src/old_value.rs

```diff
@@ -61,6 +61,16 @@ impl OldValueCache {
             miss_none_count: 0,
         }
     }
+
+    pub(crate) fn resize(&mut self, new_capacity: ReadableSize) {
+        CDC_OLD_VALUE_CACHE_MEMORY_QUOTA.set(new_capacity.0 as i64);
+        self.cache.resize(new_capacity.0 as usize);
```

Comment:
The current implementation of `resize` is not compatible with old value cache (It uses `capacity` as a length), better to leave a TODO here and support it later.

Reply:
Sorry, I didn't understand what you meant? Could you explain it in detail?

Reply:
`OldValueCache` uses a [customized policy](https://github.com/tikv/tikv/blob/master/components/cdc/src/old_value.rs#L21) to trace the size of cache.But `resize` doesn't follow this policy and just uses `capacity` as a bound of entries count in the cache, not as a size of all entries.

So we should not use `resize` on `OldValueCache` unless we fix the implementation of `resize`.

Reply:
Fixed in https://github.com/tikv/tikv/pull/10651/commits/834319750ec2fd127f10552d40f3f9f0f693e901